modules/aws-outposts-ingress-controller-fix-subnet.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-aws-outposts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="aws-outposts-ingress-controller-fix-subnet_{context}"]
+= Ensuring the default Ingress Controller uses the correct subnet
+
+When you extend an AWS VPC cluster to an Outpost, you must ensure that the default Ingress Controller uses the cloud-based subnets and not the Outpost subnet.
+To prevent it from using the Outpost subnet, annotate the default Ingress Controller service with the cloud-based subnets.
+
+[NOTE]
+====
+Manually ensuring that the default Ingress Controller uses the correct subnets is a workaround for clusters installed with {product-title} {product-version}.
+====
+
+.Prerequisites
+
+* You have installed an {product-title} cluster into a custom VPC on AWS.
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+.Procedure
+
+* Annotate the publishing Classic Load Balancer service of the default Ingress Controller with the cloud-based subnets by running the following command:
```

Comment:
In most of the OpenShift clusters the default ingress controller's load balancer has more than 1 subnet (1 for each availability zone) for the purpose of higher availability. However it's not strictly necessary as the CLB can be created with 1 subnet too.
```suggestion
* Annotate the publishing service of the default Ingress Controller with the cloud-based subnet(s) by running the following command:
```

Reply:
ACK - we don't use `noun(s)`, but can use "any" to accomplish the same effect.